models/src/main/java/com/vimeo/networking2/TeamPermissionCurrentPermissions.kt

```diff
@@ -0,0 +1,14 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Represents the current permission for a [TeamPermission]. This only contains the uri for said permission as denoted
+ * by [permissionPolicyUri]
+ */
+@JsonClass(generateAdapter = true)
+data class TeamPermissionCurrentPermissions(
```

Comment:
Needs a `param` doc for `permissionPolicyUri`.

Reply:
Once again, this is inlined int he description, but I'll move it out to a param.